.eslintrc.js

```diff
@@ -34,6 +34,87 @@ module.exports = {
     'ember/no-empty-attrs': 2,
     'ember/no-jquery': 2,
     'ember/no-observers': 2,
+
+    'ember/order-in-models': [
+      2,
+      {
+        order: ['attribute', 'relationship', 'single-line-function', 'multi-line-function'],
+      },
+    ],
+
+    'ember/order-in-components': [
+      2,
+      {
+        order: [
+          'service',
+          'property',
+          'empty-method',
+          'single-line-function',
+          'multi-line-function',
+          'observer',
+          'init',
+          [
+            // all the members of this array will be treated equally
+            'didReceiveAttrs',
+            'willRender',
+            'willInsertElement',
+            'didInsertElement',
+            'didRender',
+            'didUpdateAttrs',
+            'willUpdate',
+            'didUpdate',
+          ],
+          'method',
+          'actions',
+          ['willDestroyElement', 'willClearRender', 'didDestroyElement'], // all the members of this array will be treated equally
```

Comment:
Why are these equal, why not have them in the order `willClearRender`, `willDestroyElement`, `didDestroyElement` (i.e. execution order)?

Reply:
no particular reason, we could stick to default order 🙂